modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/dev/entry.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {render} from '@liferay/frontend-js-react-web';
+
+import Diagram from '../src/main/resources/META-INF/resources/js/Diagram';
+
+import '../src/main/resources/META-INF/resources/css/diagram.scss';
+
+render(
+	Diagram,
+	{
+		imageURL: './assets/lfr_diagram_engine.png',
+		namespace: 'portlet_shop_by_diagram_namespace_',
+		spritemap: './assets/clay/icons.svg',
```

Comment:
Just out of curiosity, is there any reason  we're adding `spritemap` here? I'm asking because it's already [there](https://github.com/liferay/liferay-portal/blob/575e1b9c8ce139ee7543f4b53e6cd8edc5d544ef/modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/render.tsx#L89) by default

Reply:
yes, because
`<ClayIconSpriteContext.Provider value={spritemap}>
<ClayIcon symbol="add-cell" />
    </ClayIconSpriteContext.Provider>`
needs a value or am i wrong? Are you showing me the spritemap's svg it's added in the overridden render function that we use importing frontend-js-react-web?